tests/integration_tests/test_examples.py

```diff
@@ -1,39 +1,84 @@
 """
 This is an integration test to make sure scripts from examples/
-work when running `python examples/xx/xxx.py`.
+work when executing `./examples/[*/]*.py`.
 """
-from garage.experiment import LocalRunner, run_experiment
-from garage.np.baselines import LinearFeatureBaseline
-from garage.tf.algos import VPG
+import os
+import subprocess
+import tempfile
+from unittest.mock import Mock
+
+import pytest
+
+from garage.envs import GridWorldEnv
+from garage.np.policies import ScriptedPolicy
 from garage.tf.envs import TfEnv
-from garage.tf.policies import CategoricalMLPPolicy
 
+examples_root_dir = 'examples/'
```

Comment:
i don't think this is really necessary. if someone moves the examples around in the repo they will need to update this string regardless. in any case, it should be ALL_CAPS